include/actuator/ActuatorLineFASTNgp.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
```

Comment:
It doesn't look like this or the associated *.C file are used.  Please delete them.